usautobuild/actions/builder.py

```diff
@@ -165,6 +166,7 @@ def build(self, target: str) -> None:
 
     def start_building(self) -> None:
         log.info("Starting a new build: %s", git_version(directory=self.config.project_path, brief=False))
+        time_start = time.time()
```

Comment:
inconsistent naming - time_start/build_time. I suggest at least changing it to start_time here